scenarios/fuel_test/contracts/integration-tools/src/main.rs

```diff
@@ -0,0 +1,56 @@
+extern crate dotenv;
+
+use dotenv::dotenv;
+use fuels::prelude::*;
+use fuels::types::ContractId;
+use std::env;
+use std::str::FromStr;
+
+abigen!(Contract(
+    name = "AllEvents",
+    abi = "../all-events/out/debug/all-events-abi.json"
+),);
+
+const ALL_EVENTS_CONTRACT: &str = "0x9a719b1ff9db28f1493375b06ad0195370786e217473eacc4be973ecd6281420";
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // Load .env file
+    dotenv().ok();
+
+    let phrase = env::var("MNEMONIC").expect("MNEMONIC must be set in .env");
+
+    let provider = Provider::connect("testnet.fuel.network").await.unwrap();
```

Comment:
Where does this get run btw? Or is it just for local testing?

Reply:
It's for local testing. I previously used the Sway Playground, but they removed the ability to call methods from there 😢